src/services/fileStorage/model.js

```diff
@@ -107,14 +106,18 @@ const fileSchema = new Schema({
 
 enableAuditLog(fileSchema);
 
+/*
+query list with bigges impact of database load
+schulcloud.files               find         {"name": 1, "parent": 1}  -> 1
+*/
+fileSchema.index({ name: 'text', parent: 1 }); // ok = 1
+fileSchema.index({ creator: 1 }); // ?
 // make file-model searchable
-fileSchema.index({ name: 'text' });
-
+fileSchema.index({ name: 'text' }); // ?
```

Comment:
i am not sure about this... zthis index can get quite big. instead we should check If we can optimize the search

Reply:
We let it with single parent and single text index, but remove the direct query { parent: text, parent: 1 }